_rules/iframe-accessible-name-cae760.md

```diff
@@ -150,12 +151,22 @@ Does not apply to non `iframe` element.
 
 #### Inapplicable Example 2
 
-`iframe` is not [included in the accessibility tree][].
+This `iframe` is not part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.
```

Comment:
```suggestion
This `iframe` is neither part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.
```